pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -287,3 +302,7 @@ func (e *BootstrapSigner) getTokens() map[string]string {
 
 	return ret
 }
+
+func (e *BootstrapSigner) getConfigMapKey() string {
```

Comment:
we can use func like `cache.DeletionHandlingMetaNamespaceKeyFunc` get configmap key?

Reply:
@zjj2wry yeah, we can. But `e.configMapNamespace` and `e.configMapName` here is always not changed, so we can simply  the logic

Reply:
got it, this lgtm